src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputPromptDefinition.java

```diff
@@ -0,0 +1,84 @@
+package org.jenkinsci.plugins.workflow.support.steps.input;
+
+import hudson.Util;
+import hudson.model.ParameterDefinition;
+import org.apache.commons.lang.StringUtils;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Holds the information needed to query for input
+ */
+public class InputPromptDefinition {
+    private final String message;
+
+    private String id;
+
+    private String submitter;
+
+    private List<ParameterDefinition> parameters = Collections.emptyList();
+
+    private String ok;
+
+    public InputPromptDefinition(String message) {
```

Comment:
:bug: this constructor appears unused, so better to delete it. (Better still to delete the whole class, as mentioned below.)